tachyon/math/geometry/point2.h

```diff
@@ -8,6 +8,8 @@
 
 namespace tachyon::math {
 
+// |Point2| represents points in (x,y) coordinates.
+// Affine form of an elliptic curve point is represented as |Point2|
```

Comment:
Can we explain this in a uniform way?

```suggestion
// |Point2| represents points expanded into 2-element vectors.
// It can represent elliptic curve points in Affine form.
```